LondonBritishLibrary/orient/BLorient799.xml

```diff
@@ -60,61 +124,124 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">219</measure>
-                                    
-                                    
-                                    
-                                    
-                                    
-                                    <dimensions type="outer" unit="mm">
-                                        <height>343</height>
-                                        <width>321</width>
+
+                                    <dimensions type="outer" unit="in">
+                                        <height>13.5</height>
+                                        <width>12.625</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
                             <layoutDesc>
-                                <layout columns="2" writtenLines="22" cert="high">
+                                <layout columns="3" writtenLines="28 31" cert="high">
                                 </layout>
                             </layoutDesc>
-                            
-                            
+
+
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript was written by three scribes, <locus from="5" to="26"/>, <locus from="27" to="87"/> and <locus from="89" to="214"/>, all of the earlier part of 
+                                <desc>The manuscript was written by three scribes, <locus from="5" to="26"/>, <locus from="27" to="87"/> and <locus from="89" to="214"/>, all of the earlier part of
                                     the <date>18th century</date>, the third portion being dated <date when="1706"/>
```

Comment:
If there are three scribes, there should be three handNotes, because each hand receives its id. The hands should be associated with the folia/ msItems. See examples in the GL